solr/benchmark/src/java/org/apache/solr/bench/generators/Lists.java

```diff
@@ -32,17 +31,13 @@ public final class Lists {
   private Lists() {}
 
   static <T> Gen<List<T>> listsOf(Gen<T> generator, Gen<Integer> sizes) {
-    return listsOf(generator, arrayList(), sizes).mix(listsOf(generator, linkedList(), sizes));
```

Comment:
was going to comment on this, because of the removal of `.mix`, but I think what you are saying is that the `.mix` doesn't actually do anything/matter?

Reply:
I'm going to revert these benchmark changes since doesn't seem to be necessary. https://github.com/quicktheories/QuickTheories/blob/master/core/src/main/java/org/quicktheories/core/Gen.java#L168 btw says mix is just combining the links.